ajax/libs/arraysweeper/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "arraysweeper",
+  "filename": "arraysweeper.min.js",
+  "version": "0.1.0",
+  "description": "A array based implementation of a minesweeper board. Can be used via a browserconsole or the command line via node JS. The main intended purpose of this is to provide the board, and logic to implement a traditional minesweeper game using the api.",
+  "homepage": "https://github.com/arschmitz/arraysweeper/",
+  "keywords": [
+    "array",
+    "sweeper",
+    "mine",
+    "sweeper",
+    "board",
+    "game"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/arschmitz/arraysweeper.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "arraysweeper.min.js"
```

Comment:
You can use the file rule `"**/*"` inside the `dist` directory to include all files instead of `"arraysweeper.min.js"`.